easybuild/easyconfigs/p/Python/Python-3.3.2-foss-2016.04.eb

```diff
@@ -0,0 +1,45 @@
+name = 'Python'
+version = '3.3.2'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'foss', 'version': '2016.04'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
```

Comment:
@AFCJamie same here, please drop the opt and optarch entries here, they're enabled by default